Update for new Twitch player

MattiasBuelens opened the pull request:
Twitch recently rolled out an update to the video player. The new player uses HTML/CSS/JavaScript for the player UI, although the actual playback is still handled by Flash.

Most importantly, the player controls now appear in an overlay on top of the video instead of in a bar below it. This means that the adjustment for the bar height is no longer needed when positioning the touch overlay.

![newtwitchplayerui](https://cloud.githubusercontent.com/assets/649348/8823307/1fe0d952-306c-11e5-85c1-9f918480572f.PNG)

I'm not sure if this update has been rolled out to _all users_ immediately, or is being gradually rolled out to a part of the users. In the latter case, we'd need some extra detection to determine _if_ the bar height should be subtracted or not. I also haven't tested this with BetterTTV yet, not sure how that affects this.

MattiasBuelens commented:
Just asked in IRC if they see the new player UI yet, and it looks like it hasn't been fully rolled out yet. So we **will** need some detection first.

lostcoaster commented:
Thank you for the work. But the player has not been changed for me. 

Can you give me a copy of the updated html file? I will see how the change can be detected.

MattiasBuelens commented:
It's okay, I can get the old player in a different browser. I'll whip up some detection logic later today.

MattiasBuelens committed:
Update for new Twitch player.

Player controls are now in an overlay on top of the video, so the bar height is no longer needed for positioning.
Uses the presence of the new bottom controls element to determine whether the new player is enabled.

MattiasBuelens commented:
Now detecting whether there's a `#player .player-controls-bottom` element before changing the bar height in the calculation. Works with both the old and new player.
